test/specs/components/Image/Image-test.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import { isConformant, handlesAccessibility, getRenderedAttribute } from 'test/specs/commonTests'
+
+import Image from 'src/components/Image/Image'
+import { MenuBehavior } from 'src/lib/accessibility'
+import { getTestingRenderedComponent } from 'test/utils'
+
+describe('Image', () => {
+  isConformant(Image)
+
+  describe('Image accessibility', () => {
+    handlesAccessibility(Image, {
+      defaultRootRole: undefined,
+      accessibilityOverride: MenuBehavior,
+      overriddenRootRole: 'menu',
+    })
+
+    test('set to true if alt is not defined', () => {
```

Comment:
from test case's text it is unclear what is set to `true` - only from test's implementation code there is a hint that it is about `aria-hidden` attribute